msibi/state.py

```diff
@@ -62,15 +67,32 @@ def save_runscript(self, table_potentials, table_width, engine='hoomd',
         """Save the input script for the MD engine. """
 
         # TODO: Factor out for separate engines.
-        header = list()
-        header.append(HOOMD_HEADER.format('start.xml', self.kT, table_width))
-        for type1, type2, potential_file in table_potentials:
-            header.append(HOOMD_TABLE_ENTRY.format(**locals()))
-        header = ''.join(header)
-        with open(os.path.join(self.state_dir, runscript)) as fh:
-            body = ''.join(fh.readlines())
-
-        runscript_file = os.path.join(self.state_dir, 'run.py')
-        with open(runscript_file, 'w') as fh:
-            fh.write(header)
-            fh.write(body)
+
+        if engine.lower() == 'hoomd':
```

Comment:
`engine.lower()` isn't necessary anymore. I think this is still present in a couple of places but since it's being processed only upon initializing the MSIBI class, we never need to convert to lower case anymore